src/styles/blocks/page.scss

```diff
@@ -0,0 +1,52 @@
+.page {
+  scroll-behavior: smooth;
+
+  &__body {
+    margin: 0;
+    font-family: "Cinzel", serif;
```

Comment:
It's a good practice to have a fallback font in case the specific font is not loaded. Add a generic font family like 'serif' or 'sans-serif' after the 'Cinzel' font.